app/views/lesson_plans/show.html.erb

```diff
@@ -1,39 +1,37 @@
 <h1>Lesson Planner</h1>
 
 <% if @lesson_plan_lessons.empty? %>
-  <p>There's nothing here! <%= link_to "Browse our training topics ", topics_path %> to start adding lessons to your plan.</p>
```

Comment:
If you have time could you undo these whitespace changes? Also btw, with regards to below, in erb templates stuff inside a ruby block is typically indented another level by convention. So for example:

```html
<% if blah blah blah %>
  <div class="something">or another</div>
<% end %>

<ol>
  <% lessons.each do |lesson| %>
    <li>
      <%= lesson.name %>
    </li>
  <% end %>
</ol>
```

Reply:
Atom seems to want to mess up the spacing, I'll fix it manually.